week3/61-sistiema-linieinykh-uravnienii-1/task.py

```diff
@@ -0,0 +1,20 @@
+def safe_to_int(n):
+    if n.is_integer():
+        return int(n)
+    else:
+        return n
+
+
+def main():
+    a, b, c, d, e, f = float(input()), float(input()), float(
+        input()), float(
+        input()), float(input()), float(input())
+    if a == 0:
+        a, b, c, d, e, f = c, d, a, b, f, e
```

Comment:
no! do not reuse same variables - code is unreadable. 

use a11, a12 and so on. add comment with initial system of equations and a11, a12 .. one - explane new system of equations constraint (that a11 is always >0)

Reply:
done